app.rb

```diff
@@ -57,32 +53,30 @@ def create_book
 
   def create_rental
     puts 'Choose a book from the list by number'
-    @books.each_with_index do |book, index|
-      puts "#{index} - #{book.title} by #{book.author}"
-    end
+    @books.each_with_index { |book, index| puts "#{index} - #{book.title} by #{book.author}" }
     book_index = gets.chomp.to_i
     book = @books[book_index]
     puts 'Choose a person from the list by number:'
-    @people.each_with_index do |person, index|
-      puts "#{index} - #{person.name} - #{person.age} - #{person.class}"
-    end
+    @people.each_with_index { |person, index| puts "#{index} - #{person.name} - #{person.age} - #{person.class}" }
     person_index = gets.chomp.to_i
     person = @people[person_index]
-    puts 'Date: '
+    puts 'Date in format yyyy/mm/dd:'
     date = gets.chomp
     Rental.new(date, book, person)
     puts 'Rental created!'
   end
 
```

Comment:
* Kindly ensure all requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input. as SOLID principle is to be followed [Check here for more on SOLID principle](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid`)